virt/virtual_machines/virtual_disks/virt-configuring-shared-volumes-for-vms.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="virt-configuring-shared-volumes-for-vms"]
+= Configuring shared volumes for virtual machines
+include::_attributes/common-attributes.adoc[]
+:context: virt-configuring-shared-volumes-for-vms
+
+toc::[]
+
+
+To allow multiple virtual machines (VMs) to share the same underlying storage, you can configure shared disks. A shared disk's volume must be block mode.
```

Comment:
Not necessary, this could be a raw image on a filesystem PVC. In this case, two VMs could share the same disk images

Reply:
@alicefr, can you elaborate a bit on what you mean here?

Reply:
The `shareable` option could be used also on a filesystem PVC. In this case, 2 VMs could access the same raw image on the same filesystem PVC.